ideahacks/helpers/userFunctions.js

```diff
@@ -0,0 +1,31 @@
+const formatUser = user => {
+  if (user.hasTeam) {
```

Comment:
Are you sure these two if statements are correct? It looks like if user.hasTeam is "true", then both user.hasATeam and user.hasNoTeam will be both be defined. Did you mean to protect the second part with an else statement? Did you test if this worked as is?